sormas-backend/src/main/java/de/symeda/sormas/backend/immunization/Immunization.java

```diff
@@ -312,4 +318,13 @@ public Country getCountry() {
 	public void setCountry(Country country) {
 		this.country = country;
 	}
+
+	@OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, mappedBy = VaccinationEntity.IMMUNIZATION, fetch = FetchType.EAGER)
```

Comment:
What's the reason for using eager fetching here?

Reply:
I remember very distinctly that there were tests failing because of precisely this. Seems like that is no longer the case though, as the tests passed now even when I removed it again.

Reply:
Good to hear - the parameter is still there though :)

Reply:
Turns out, that was also the one which makes the tests fail if removed ;)

Specifically, it caues a `org.hibernate.LazyInitializationException` in two tests of `ImmunizationFacadeEjbTest` - apparently it has something to do with the session configuration. There is probably a better way to solve this, but I am not very experienced in hibernate in that regard. Do you have any ideas?